fs/dafsclient/rmtfile.hpp

```diff
@@ -66,8 +66,6 @@ extern DAFSCLIENT_API void disconnectRemoteIoOnExit(IFileIO *fileio,bool set=tru
 extern DAFSCLIENT_API bool resetRemoteFilename(IFile *file, const char *newname); // returns false if not remote
 
 
-extern DAFSCLIENT_API void enableAuthentication(bool set=true); // default enabled for clients, disabled for server
-
```

Comment:
ok to remove entry point ?  Should it be a no-op ?  Does any client link to this lib that we should be concerned with ?

Reply:
I think fine to remove, only other (same version) platform dlls link to this.